lib/logger/lib/logger.ex

```diff
@@ -973,6 +973,13 @@ defmodule Logger do
         # as map
         Logger.#{level}(#{inspect(Map.new(report))})
 
+    Report message with metadata (maps or keywords):
+
+        # as a keyword list
+        Logger.#{level}("this is a #{level} message", #{inspect(report)})
+
+        # as map
+        Logger.#{level}("this is a #{level} message", #{inspect(Map.new(report))})
```

Comment:
To avoid confusion we should probably choose a different example for the metadata. Maybe `user_id: 13` or `request_id: "xU32kFa"`?